url-updater/src/main/java/com/devonfw/tools/ide/url/updater/helm/HelmUrlUpdater.java

```diff
@@ -23,11 +29,13 @@ protected String getGithubOrganization() {
   @Override
   protected void addVersion(UrlVersion urlVersion) {
 
-    String baseUrl = "https://get.helm.sh/helm-${version}-";
+    String version = urlVersion.getName();
+    String baseUrl = "https://get.helm.sh/helm-v${version}-";
     doAddVersion(urlVersion, baseUrl + "windows-amd64.zip", WINDOWS);
     doAddVersion(urlVersion, baseUrl + "linux-amd64.tar.gz", LINUX);
     doAddVersion(urlVersion, baseUrl + "darwin-amd64.tar.gz", MAC);
-    doAddVersion(urlVersion, baseUrl + "darwin-arm64.tar.gz", MAC, ARM64, "");
+    if (isVersionGreaterThan(version, "3.4.0"))
+      doAddVersion(urlVersion, baseUrl + "darwin-arm64.tar.gz", MAC, ARM64,"");
```

Comment:
In Java it is best practice to ALWAYS use curly braces for `if` statements. Why?
Look at the following valid Java example:
```
public void myMethod() {
  if (false)
    System.out.println("Inconsistentcy detected: false is true");
    System.exit(-1);
  System.out.println("All is sane here");
}
```

Due to the indentation the developer might have assumed that the second statement (`System.exit`) still belongs to the `if` block but it does not and the code does not behave as expected. This is a common pitfall when extending existing code. It will always get clear and obvious if your code looks like this (also the wrong indentation):

```
public void myMethod() {
  if (false) {
    System.out.println("Inconsistentcy detected: false is true");
  }
    System.exit(-1);
  System.out.println("All is sane here");
}
```

We have created a formatter for Eclipse in devonfw-ide that automatically aligns and formats your code whenever you save it resolving all such anti-patterns. Are you using IntelliJ oder VisualStudioCode? Maybe we can invest to get the same autoformatter preconfigured for your favorite IDE....